### DISCUSSION TITLE
Third View Mode - Display Only the HTML Preview

### DISCUSSION THREAD

**Discussion by kenclark75** [KDE/ghostwriter#689] [Ideas]
I recently began using Markdown to maintain a database of text notes. I am currently evaluating Ghostwriter as my editor.
Almost all of my initial impressions are favourable with one possible exception.
There are currently only two View options. I can view editing window alone or with the "Preview in HTML" alongside it.  This is perfect when I am writing notes.
However, on more than 90% of the occasions when I use Ghostwriter, I only wish to read an existing note, without performing any edits.
On these occasions, I only need to see the Preview in HTML. The editing display is a distraction and occupies unnecessary screen space.
Would it be possible to offer a third  viewing option where only the HTML preview is displayed?

**Comment by mdnava** [comment]
Please consider adding this feature.. a third view mode to display HTML Preview without editing mode.

And possibly a command line option to start in HTML Preview mode.

Is currently the only reason that prevents me from using it as default markdown reader.

Best regards!

**Comment by James-William-Kincaid-III** [comment]
Same!  I just downloaded Ghostwriter and did a search to find out what I might be missing in the UI, and ended up on this page.  This should have been a foundational feature!  GEESH! 😁😁

**Comment by JahelCuadrado** [comment]
I just downloaded the program, and this is the first thing I'm missing—being able to view only the HTML. If you add these three things, for me, it would be a perfect 10/10 program:

1 - HTML-only view
2 - Dark mode affecting the title bar
3 - Ability to hide the menu bar

**Comment by Yzrsah** [comment]
Lack of HTML view is the #1 problem with this app.

I simply want to read my Markdowns sometimes, not edit them.

There's too much stuff on-screen, so I have to use a different Markdown editor just for reading.
This made me uninstall ghostwriter because it was not doing its basic job and therefore became redundant.

**Comment by da-astro** [comment]
Using v2.1.1

If you have both code and html panes open, just slide the panel separator (on the left side for left-to-right readers) until the html pane covers the code pane. Code pane is still underneath, but the effect is the same.

**Reply by yason-fi** [reply]
This dragging doesn't seem to work on 25.04.0. But really, there's a button to hide the HTML preview. I could really do with another button to hide the editor as well. I often read markdown documents first and then maybe edit them, so being able to start in "read-only" mode would be really useful, here's my vote. Just hide the editor pane.

**Comment by mdnava** [comment]
Well, five years have passed since this issue was first opened. 
I ended up buying a license for **Typora** to solve this issue.

I do think Ghostwriter is a great program that would benefit from having a WYSIWYG mode, or just what's been asked since 2021: 

- The option to have the HTML Preview as default mode to read Markdown files.
- A button for this "Third View Mode", and the configuration option to select it as default mode if the users wants it.

Best regards!
